app/src/main/kotlin/com/intive/picover/profile/view/ProfileScreen.kt

```diff
@@ -45,17 +63,24 @@ import com.intive.picover.profile.model.Profile
 import com.intive.picover.profile.viewmodel.ProfileViewModel
 import com.skydoves.landscapist.ImageOptions
 import com.skydoves.landscapist.coil.CoilImage
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalComposeUiApi::class)
 @Composable
 fun ProfileScreen(viewModel: ProfileViewModel) {
 	val profile by viewModel.profile
 	val context = LocalContext.current
+	val keyboardController = LocalSoftwareKeyboardController.current
 	var isDeleteAccountDialogVisible by remember { mutableStateOf(false) }
 	val photoLauncher = rememberLauncherForActivityResult(ActivityResultContracts.PickVisualMedia()) { uri ->
 		if (uri != null) {
 			viewModel.updateAvatar(uri)
 		}
 	}
+	val coroutineScope = rememberCoroutineScope()
+	val sheetState = rememberModalBottomSheetState()
+	var userName by remember { mutableStateOf("") }
```

Comment:
do we need another userName now, as we have one in viewmodel ?

Reply:
Now variable in VM is a state and we must probably have an extra userName on the UI to handle changing text. Maybe I don't see a way to have only one variable. 😄